nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -820,8 +819,8 @@ def find_pairs_of_nodes_with_different_width(self, pairs_of_nodes: List[Tuple[st
                          f'Width: {start_width} vs {end_width}. Shapes: {start_output_shape} vs {end_output_shape}'
             else:
                 continue
-            nncf_logger.warning('The block [\n\t{},\n\t{}\n]\n can`t be skipped, because {}'.format(
-                start_node_name, end_node_name, reason))
+            nncf_logger.warning(
```

Comment:
Propose moving to debug
@jpablomch

Reply:
Moving to debug works for the user. Thanks @ljaljushkin!

Reply:
Will fix in a follow-up PR

Reply:
#1431